packages/autocomplete-plugin-algolia-insights/src/types/InsightsClient.ts

```diff
@@ -1 +1,30 @@
-export type InsightsClient = any;
+import type {
+  InsightsMethodMap,
+  InsightsClient as _InsightsClient,
+} from 'search-insights';
+
+export type {
+  Init as InsightsInit,
+  AddAlgoliaAgent as InsightsAddAlgoliaAgent,
+  SetUserToken as InsightsSetUserToken,
+  GetUserToken as InsightsGetUserToken,
+  OnUserTokenChange as InsightsOnUserTokenChange,
+} from 'search-insights';
```

Comment:
this makes search-insights a dependency (optional/peer) right?

Reply:
It's already the case no? https://github.com/algolia/autocomplete/blob/next/packages/autocomplete-plugin-algolia-insights/package.json#L41